docs/reports/migrate_doid.md

```diff
@@ -0,0 +1,686 @@
+## DOID
+[Interactive FlatGithub table](https://flatgithub.com/monarch-initiative/mondo-ingest?filename=src/ontology/slurp/doid.tsv)
```

Comment:
@matentzn @hrshdhgd @sabrinatoro @nicolevasilevsky Check this out! Added links to the new "FlatGitHub" in documentation pages for each ontology in slurp/migrate, mapping progress monitor, and "deprecated terms in mondo" pages.

e.g. `docs/reports/migrate_doid.md` now has this link at the top of the page:
[Interactive FlatGithub table](https://flatgithub.com/monarch-initiative/mondo-ingest?filename=src/ontology/slurp/doid.tsv)

<img width="1040" alt="Screen Shot 2023-04-02 at 4 14 37 PM" src="https://user-images.githubusercontent.com/13045020/229377006-e21d65ce-608d-421f-8d08-d38c4dc1511f.png">

Reply:
That is _so_ cool! It would be nice to get this for the `lexmatch` results as well :) But for now, great thinking!